templates/ignition.yaml

```diff
@@ -0,0 +1,184 @@
+ignition:
+  version:
+   major: 2
+   minor: 0
+   patch: 0
+systemd:
+  units:
+    - name: etcd2.service
+      enable: false
+      mask: true
+    - name: fleet.service
+      enable: false
+      mask: true
+    - name: systemd-modules-load.service
+      enable: true
+    - name: systemd-networkd-wait-online.service
+      enable: true
+    - name: docker.service
+      enable: true
+    - name: var-lib-docker.mount
+      enable: true
+      contents: |
+        [Unit]
+        Description=Mount /dev/sda to /var/lib/docker
+        Before=docker.service
+        [Mount]
+        What=/dev/sda
+        Where=/var/lib/docker
+        Type=xfs
+        [Install]
+        WantedBy=multi-user.target
+    - name: docker.service
+      dropins:
+        - name: 40-docker.conf
+          contents: |
+            [Unit]
+            Requires=docker.socket var-lib-docker.mount
+            After=docker.socket var-lib-docker.mount
+            [Service]
+            Environment="DOCKER_OPTS=--iptables=true --storage-driver=overlay --log-opt max-size=50m --log-opt max-file=2 --live-restore --userland-proxy=false --icc=false --disable-legacy-registry=true {{index .TemplatesEnv "docker_args"}}"
+            Environment="DOCKER_OPT_BIP="
+            Environment="DOCKER_OPT_IPMASQ="
+            Environment="DOCKER_OPT_MTU="
+    - name: ensure-connectivity.service
+      enable: true
+      contents: |
+        [Unit]
+        Description=Ensures theres connectivity before starting etcd2
+        After=systemd-networkd.service
+        Wants=systemd-networkd.service
+        Before=etcd3.service
+
+        [Service]
+        Type=oneshot
+        ExecStart=/bin/sh -c 'while [[ -z $(curl -k -s {{index .TemplatesEnv "mayu_https_endpoint"}}) ]] ; do sleep 2 ; done'
+        RemainAfterExit=yes
+
+        [Install]
+        WantedBy=multi-user.target
+    - name: set-environment.service
+      enable: true
+      contents: |
+        [Unit]
+        Description=Configures giantswarm environment
+        Wants=network-online.target
+        After=network-online.target
+
+        [Service]
+        Type=oneshot
+        ExecStartPre=/bin/sh -c "hostnamectl set-hostname machine-{{.Host.Hostname}}"
+        ExecStartPre=/bin/sh -c 'echo "SERIAL={{.Host.Serial}}" > /etc/mayu-env'
+        ExecStart=/bin/sh -c 'echo "COREOS_PRIVATE_IPV4={{.Host.InternalAddr}}" > /etc/environment'
```

Comment:
do we still need that at all? afaik no unit is using that anymore.

Reply:
removed